soccer/src/soccer/joystick/sdl_manual_control.cpp

```diff
@@ -56,7 +56,11 @@ ControllerCommand KeyboardController::get_command() const {
         command.kick_power = kick_power_;
     }
 
-    command.dribble_power = dribble_power_;
+    if (key_down(keystate, SDL_SCANCODE_SPACE) || key_down(keystate, SDL_SCANCODE_RSHIFT)) {
+        command.dribble_power = 255.0;  // max is 255
+    } else {
+        command.dribble_power = dribble_power_;
```

Comment:
I feel instead that we should set dribble_power_ = 255 in the header. And (use that on line 60). Here should just set to 0.0 for clarity.